.github/workflows/_python-tests.yml

```diff
@@ -129,6 +129,26 @@ jobs:
         env:
           PROPERTY_TESTING_PRESET: ${{ inputs.property_testing_preset }}
 
+  test-thin-client-async:
```

Comment:
should we just make this the same as the job above with a new step or similar rather than adding another job.

Reply:
yep, not sure why I originally split it